examples/torch/common/models/classification/mobilenet_v2_cifar10.py

```diff
@@ -0,0 +1,154 @@
+"""
```

Comment:
Just in case. You can use 3rd party projects for CIFAR-10 instead of having your own implementation. Check this one: https://github.com/chenyaofo/pytorch-cifar-models

```
import torch

model = torch.hub.load("chenyaofo/pytorch-cifar-models", "cifar10_mobilenetv2_x1_0", pretrained=True)
```

Reply:
Actually, we use 3rd party project here: https://github.com/huyvnphan/PyTorch_CIFAR10.
Looks like the suggested approach with `torch.hub` is not applicable for this repository, because weights are downloaded for all models once: https://github.com/huyvnphan/PyTorch_CIFAR10/blob/master/data.py#L21

The results in the paper were achieved with these models and weights, that's why switching to the new suggested  repository will require re-running all experiments with mobilenet and resnet and potentially will require changing of some hyperparameters.

@jpablomch  



https://github.com/huyvnphan/PyTorch_CIFAR10/blob/master/data.py#L21